constants/constants.F90

```diff
@@ -37,12 +37,24 @@ module constants_mod
 
 real :: realnumber !< dummy variable to use in HUGE initializations
 
+#ifdef SMALL_EARTH
+#if defined(DCMIP) || (defined(HIWPP) && defined(SUPER_K))
+ real, public, parameter :: small_fac =  1._r8_kind / 120._r8_kind #only needed for supercell test
+#elif defined(HIWPP)
+ real, public, parameter :: small_fac = 1._r8_kind / 166.7_r8_kind
+#else
+ real, public, parameter :: small_fac = 1._r8_kind / 10._r8_kind
+#endif
+#else
+ real, public, parameter :: small_fac = 1._r8_kind
```

Comment:
1. `small_fac` should have some kind of documentation to tell us what it is.  I realize it wasn't there before, but these values are very specific
2. Can `small_fac` have more digits?  It seems like the rounding of these numbers is left to the compiler instead of the programmer.  That would change the accuracy.

Reply:
I will add documentation on what small_fac does.

Reply:
With the exception of those that are strictly typed, precision of all constants in this file is set at compile time.  small_fac is defined to be consistent with the parameters it is modifying.